packages/usa-modal/src/index.js

```diff
@@ -122,13 +131,13 @@ function toggleModal(event) {
     body.classList.toggle(PREVENT_CLICK_CLASS, safeActive);
   }
 
-  // Account for content shifting from body overflow: hidden
-  // We only check paddingRight in case apps are adding other properties
-  // to the body element
-  body.style.paddingRight =
-    body.style.paddingRight === TEMPORARY_PADDING
-      ? INITIAL_PADDING
-      : TEMPORARY_PADDING;
+  // Temporarily increase body padding to include the width of the scrollbar.
+  // This accounts for the content shift when the scrollbar is removed on modal open.
+  if (body.style.paddingRight === TEMPORARY_BODY_PADDING) {
+    body.removeAttribute("style");
```

Comment:
Is it possible to only reset `paddingRight` instead of removing styles altogether?

Reply:
Updated `removeAttribute` to `removeProperty` in https://github.com/uswds/uswds/pull/5493/commits/3fac64c4b43b71242f4b5f87e11c27725a944071